recipes/taurus/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "taurus" %}
+{% set version = "4.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5329036608b94444f3512c89d493092fb658d069bcd4d1f82f393e70fdefc3d9
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  run:
+    - {{ pin_subpackage('taurus-core', max_pin="x.x.x.x") }}
+    - {{ pin_subpackage('taurus-qt', max_pin="x.x.x.x") }}
+
+outputs:
+  - name: taurus-core
+    script: build_core.sh
```

Comment:
script: {{ PYTHON }} -m pip install . --no-deps -vv

Reply:
I tried that before but got:
```
Attempting to finalize metadata for taurus-core
INFO conda_build.metadata:finalize_outputs_pass(748): Attempting to finalize metadata for taurus-core
Error: Failed to render jinja template in /home/conda/staged-recipes-copy/recipes/taurus/meta.yaml:
'PYTHON' is undefined
```

This is why I used an external script.

Reply:
I just pushed that change anyway to check this wasn't an issue on my machine (even if I used `./.scripts/run_docker_build.sh`).

Reply:
Same error. Am I doing something wrong?
Or should that be reported as a conda-build bug?

Reply:
I reverted my change to go back to my initial commit. Seems like a file has to be used for `script` under `outputs`.